src/main.rs

```diff
@@ -210,7 +217,13 @@ Possible values: [{0}{1}{2}{3}{4}{5}{6}{7}{8}{9}{10}{11}{12}{13}{14}{15}]",
         Vec::new()
     };
 
-    let info = Info::new(&dir, custom_logo, custom_colors, disable_fields)?;
+    let bold_flag = if matches.is_present("no-bold") {
+        false
+    } else {
+        true
+    };
```

Comment:
Can be reduced to
```rust
let bold_flag = !matches.is_present("no-bold");
```

Reply:
Good catch! Can't believe I missed that 🤦‍♂